### PR TITLE
Google integration

### PR DESCRIPTION
**Pull request by Andres2D**
# Summary
* Configure the google provider with NextAuth.js
* Add google callback and redirect actions
* Add fallback user image
* Create base Menu page

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
-import { ChakraProvider } from '@chakra-ui/react'
-import '../styles/globals.css'
-import type { AppProps } from 'next/app'
+import { Session } from "next-auth";
+import { ChakraProvider } from '@chakra-ui/react';
+import '../styles/globals.css';
```
Comment (urielPalomares) [inline pages/_app.tsx:3]: Ordernar los import

File: pages/menu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import type { NextPage } from 'next';
+import { useSession } from "next-auth/react";
+import Image from 'next/image';
+
+const Menu: NextPage = () => {
+
+  const { data: session } = useSession();
+
+  console.log(session);
```
Comment (urielPalomares) [inline pages/menu/index.tsx:9]: delete console.log

File: package-lock.json [hunk 3]
```diff
@@ -13,7 +13,10 @@
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
         "framer-motion": "^7.4.0",
+        "install": "^0.13.0",
```
Comment (urielPalomares) [inline package-lock.json:16]: ignorar este file

Comment (Andres2D) [reply package-lock.json:16]: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

**Pull request merged**